### PR TITLE
Add a separate non-blocking PR check for onboarding new tests(T0).

### PR DESCRIPTION
**Pull request by yutongzhang-microsoft**
### Description of PR

We are going to onboard a batch of test cases into PR test recently. To achieve this without disrupting the existing PR test processes, we add a separate non-blocking PR checker for onboarding new tests.

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311

#### What is the motivation for this PR?
We are going to onboard a batch of test cases into PR test recently. To achieve this without disrupting the existing PR test processes, we add a separate non-blocking PR checker for onboarding new tests.

#### How did you do it?
Add a separate non-blocking PR checker for onboarding new tests

### CODE REVIEW

File: .azure-pipelines/pr_test_scripts.yaml [hunk 1]
```diff
@@ -14,11 +14,6 @@ t0:
   - bgp/test_bgpmon.py
   - cacl/test_cacl_application.py
   - cacl/test_cacl_function.py
-  - console/test_console_availability.py
-  - console/test_console_driver.py
-  - console/test_console_loopback.py
-  - console/test_console_reversessh.py
-  - console/test_console_udevrule.py
```
Comment (wangxin) [inline .azure-pipelines/pr_test_scripts.yaml:21]: Why remove these scripts from the list? Are they unstable?

**Comment by lerry-lee** [comment]
@yutongzhang-microsoft  how to make it  as a separate non-blocking PR checker? I saw that it is parallel to other kvmtest, both under test jobs

**Comment by lerry-lee** [comment]
> @yutongzhang-microsoft how to make it as a separate non-blocking PR checker? I saw that it is parallel to other kvmtest, both under test jobs

Oh I see this config, `continueOnError: true`. Please ignore.

File: .azure-pipelines/pr_test_scripts.yaml (hunk 1 above)
Comment (yutongzhang-microsoft) [reply .azure-pipelines/pr_test_scripts.yaml]: As you suggested, I add only one case as the placeholader.

**Review by wangxin — APPROVED** [review]

**Pull request merged**